lib/Yam/Agama/Pom/RebootPage.pm

```diff
@@ -22,7 +22,22 @@ sub new {
 
 sub expect_is_shown {
     my ($self, %args) = @_;
-    assert_screen($self->{tag_installation_complete}, $args{timeout});
+    my $timeout = $args{timeout};
+    my $mouse_x = 1;
+    while (1) {
+        die "timeout ($timeout) hit on during installation" if $timeout <= 0;
+        if (check_screen $self->{tag_installation_complete}, 30) {
+            return;
+        } else {
+            $timeout -= 30;
+            diag("left total await_install timeout: $timeout");
```

Comment:
s/await_install// (that is the name of the module from where we borrow this code, no need to mention here).

Reply:
changed.